src/ini_to_core_reader.cpp

```diff
@@ -15,15 +15,25 @@
 #include <boost/lexical_cast.hpp>
 #include <boost/algorithm/string/predicate.hpp>
 #include <boost/algorithm/string/trim.hpp>
+
+// ANS: If using the R package, use Rcpp to call R's file processing
+// functions. Otherwise (e.g. if building standalone Hector), fall
+// back to boost::filesystem (which needs to be installed).
+#ifdef USE_RCPP
+#include <Rcpp.h>
+#else
 #include <boost/filesystem.hpp>
+#endif
 
 #include "core.hpp"
 #include "message_data.hpp"
 #include "ini_to_core_reader.hpp"
 #include "ini.h"
 #include "csv_table_reader.hpp"
 
+#ifndef USE_RCPP
 namespace fs = boost::filesystem;
```

Comment:
Maybe just move this up with the `#include <boost/filesystem.hpp>` to keep them together.  Then maybe in the future if we want add support to use the filesystem that is part of the standard library and fall back to boost as a last resort we could just have something like:
```
#if SOME_SORT_OF_STD_FILESYSTEM_CHECK
#include <filesystem>
namespace fs = std::filesystem;
# else
#include <boost/filesystem.hpp>
namespace fs = boost::filesystem;
#endif
```

Reply:
Good idea. Addressed in e38498333c1162edcdf1e8c30ab296a260e11114.